st2tests/st2tests/fixtures/packs/test_content_version_fixture/BUILD

```diff
@@ -0,0 +1,3 @@
+python_sources(
+    dependencies=["st2tests/st2tests/fixtures/packs:test_content_version"],
+)
```

Comment:
This manually declared dependency tells pants that `fixture.py` in this directory actually needs the files from `test_content_version` (the git submodule).